libs/policyengine-api/src/policyengine_api/simulation/router.py

```diff
@@ -0,0 +1,79 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from typing import Annotated
+from sqlmodel import Session
+from policyengine_fastapi.auth import JWTDecoder
```

Comment:
__minor, blocking:__
```suggestion
from policyengine_api.fastapi.auth import JWTDecoder
from policyengine_api.fastapi.database import SessionGeneratorFactory
```